defaults/main.yml

```diff
@@ -25,6 +25,13 @@ wireguard_conf_group: "{{ 'root' if not ansible_os_family == 'Darwin' else 'whee
 wireguard_conf_mode: 0600
 
 
+######################################
+# Settings only relevant for Debian
+######################################
+
+# Set to "false" if apt repos are managed outside this role
+wireguard_debian_manage_deb_repo: "true"
+
```

Comment:
```suggestion
# Set to "no" if WireGuard binaries and tools are managed outside this role.
# In this case the role won't install the required WireGuard packages and it's
# up to you to install the required binaries before running this role!
# Currently this setting is only respected on hosts running Debian OS!
wireguard_manage_package: "yes"

```
I'd like to keep this generic. I can imagine that this is also helpful for people running other Linux distributions. I'm totally happy with the variable name but couldn't find anything better :wink: